packages/@textlint/textlint-plugin-text/test/TextProcessor-test.ts

```diff
@@ -61,7 +61,7 @@ describe("TextProcessor", function () {
     });
     context("when target is text", function () {
         it("should report error", function () {
-            return lintText("TODO: this is todo", ".txt").then((results) => {
```

Comment:
I don't know why this code works.
However, this argument looks obviously wrong.